app/assets/javascripts/settings-controller.coffee

```diff
@@ -60,35 +62,57 @@ template = """
       <option value={{code}}>{{description}}</option>
       {{/}}
     </select>
-  </div>                           
+  </div>
 </div>
 
 
 <h1 class="settings-subheader">Works in Progress</h1>
-<a><center> Here you can access all the models that have been edited</center></a>
-  
+<div class="description">Here are all the models you have made changes to in NetLogo Web.</div>
+
 <ul>
   {{#each workInProgressLinks}}
-    <li> <a href="{{url}}">{{modelTitle}}</a> <br>Date Last Modified: {{dataAccessed}}</li>
-    <br>
+    <li><a href="{{url}}">{{modelTitle}}</a> {{storageTag}} <div>{{dateAccessed}} at {{timeAccessed}}</div></li>
   {{/each}}
 </ul>
 
 """
+#Formatting Links for HTML display
 formatLinks = (wipStorage) ->
   result = []
 
   for url, model of wipStorage.inProgress
-    if model.title and model.timeStamp
-      formattedUrl = "/launch#http://" + url.replace(/^url:\/\//, '')
+    if model.title and model.timeStamp and not(url.startsWith('disk') or url.startsWith('new')) and model.version == WIP_INFO_FORMAT_VERSION
+      storageTagOutput = ""
+      storageTag = url.split(':').shift();
+      if storageTag !="url"
+        storageTagOutput = " (tag: " + storageTag + ")"
```

Comment:
Nit: Might as well use CoffeeScript's string interpolation syntax here, just to start getting in the habit: `" (tag: #{storageTag})"`